index.d.ts

```diff
@@ -213,7 +213,7 @@ export interface AxiosStatic extends AxiosInstance {
   isCancel(value: any): boolean;
   all<T>(values: Array<T | Promise<T>>): Promise<T[]>;
   spread<T, R>(callback: (...args: T[]) => R): (array: T[]) => R;
-  isAxiosError(payload: any): payload is AxiosError;
+  isAxiosError<T = any>(payload: any): payload is AxiosError<T>;
```

Comment:
AxiosError has two type parameters, I think including both of them would be better.

https://github.com/axios/axios/blob/ea0d9c69f78070dae841c66efd7263e246cab845/index.d.ts#L139

Reply:
Good point - thanks!